packages/remix/scripts/upload-sourcemaps.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+/* eslint-disable no-console */
+const SentryCli = require('@sentry/cli');
+const argv = require('yargs/yargs')(process.argv.slice(2))
+  .option('release', { type: 'string', describe: 'The release number', demandOption: true })
+  .option('urlPrefix', { type: 'string', describe: 'The url prefix for the sourcemaps' })
+  .option('buildPath', { type: 'string', describe: 'The path to the build directory' })
+  .usage('Usage: $0 --release RELEASE [--urlPrefix URL_PREFIX] [--buildPath BUILD_PATH]').argv;
+
+const RELEASE = argv.release;
```

Comment:
Can we default the release to use propose-version? (https://docs.sentry.io/product/cli/releases/#creating-releases)